types/peakvault/index.d.ts

```diff
@@ -0,0 +1,233 @@
+// Type definitions for non-npm package peakvault-browser 0.0
+// Project: https://gitlab.com/peakd/peak-vault
```

Comment:
This link doesn't seem to exist?

Reply:
I don't think the types package should be merged here until the project is actually live and testable; reviewers have no way of verifying that these types are correct without something to go on.